lib/oppen/print_stack.rb

```diff
@@ -246,7 +246,7 @@ def write(obj)
     #
     # @return [Nil]
     def indent(amount)
-      write genspace.call(amount)
+      write genspace.call(amount) if amount.positive?
```

Comment:
Going silent is a bad idea. Let's emit a warning if it's not. Check `warn`. Or should we throw?

Reply:
No in this case we want to go silent. Indent is called after each new line, if indent is 0, it is not an error/warning, we just do not need indentation.

Reply:
But if it's negative … ?

Reply:
We will raise an error if it is negative.